tests/_support/Helper/Acceptance.php

```diff
@@ -15,6 +15,11 @@ public function sendEmail($to, $subject, $body, $encoding = null)
         if (null !== $encoding) {
             $phpmailer->Encoding = $encoding;
         }
+        if (count($attachments) > 0){
```

Comment:
This `if` is useless since the `foreach` won't run any loops if the count is 0 anyway